templates/nginx/Dockerfile.template

```diff
@@ -38,10 +33,6 @@ RUN set -x \
                                                 gettext-base \
         && rm -rf /var/lib/apt/lists/*
 
-# forward request and error logs to docker log collector
-RUN ln -sf /dev/stdout /var/log/nginx/access.log \
-        && ln -sf /dev/stderr /var/log/nginx/error.log
-
 COPY stub.conf /etc/nginx/conf.d/stub.conf
```

Comment:
I thought about `stub.conf`.

Reply:
But regarding stub.conf, it wouldn't be good to enable it in case the developer wants to connect to an external exporter?

Reply:
Probably yes, but I think we can keep it disabled by default (as was before the support for exporter was added).

But, I will leave the decision to you.

Reply:
Great. So I would go for disabling by default and adding an environment variable ENABLE_STUB. If the ENABLE_STUB = 1, enable it in entrypoint. What do you think about that?

Reply:
I think we shouldn't do it now. This is mostly FR.

I recommend to remove the file from container or just leave it as it is if you prefer.

Please also revert the deleted lines:

```
# forward request and error logs to docker log collector	
RUN ln -sf /dev/stdout /var/log/nginx/access.log \	
        && ln -sf /dev/stderr /var/log/nginx/error.log	
```

Reply:
Solved. stub.conf has been removed.